homeassistant/components/plex/view.py

```diff
@@ -0,0 +1,53 @@
+"""Implement a view to provide proxied Plex thumbnails to the media browser."""
+from __future__ import annotations
+
+from http import HTTPStatus
+import logging
+
+from aiohttp import web
+from aiohttp.hdrs import CACHE_CONTROL
+from aiohttp.typedefs import LooseHeaders
+
+from homeassistant.components.http import KEY_AUTHENTICATED, HomeAssistantView
+from homeassistant.components.media_player import async_fetch_image
+from homeassistant.core import HomeAssistant
+
+from .const import DOMAIN as PLEX_DOMAIN, SERVERS
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class PlexImageView(HomeAssistantView):
+    """Media player view to serve a Plex image."""
+
+    requires_auth = False
+    name = "api:plex:image"
+
+    def __init__(self, hass: HomeAssistant, server_id: str) -> None:
+        """Initialize a media player view."""
+        self.hass = hass
+        self.server_id = server_id
+        self.url = f"/api/plex_image_proxy/{server_id}"
+        self.extra_urls = [
```

Comment:
Why use extra urls? is there a use case where you want to fetch just `/api/plex_image_proxy/{server_id}` ?